openreview/venue/process/assignment_pre_process.js

```diff
@@ -16,12 +16,20 @@ async function process(client, edge, invitation) {
   const submissionGroupId = `${venueId}/${submissionName}${submission.number}`
 
   if (!edge.ddate) {
+    const { invitations } = await client.getInvitations({ id: inviteAssignmentId });
+    const acceptLabel = invitations[0]?.content?.accepted_label?.value ?? '';
+
     const [{ edges: inviteAssignmentEdges }, { edges: assignmentEdges }] = await Promise.all([
         client.getEdges({ invitation: inviteAssignmentId, head: edge.head }),
         client.getEdges({ invitation: edge.invitation, head: edge.head })
     ])
 
-    if (quota && inviteAssignmentEdges.length + assignmentEdges.length >= quota) {
+    // Filter assignment edges to exclude the current edge.id
+    const filteredAssignmentEdges = assignmentEdges.filter(e => e.id !== edge.id)
```

Comment:
I thought that if this is a preprocess, then the edge is not posted yet?

Reply:
I see, this is the case were we update the edge in the `Invite_Assignment` process?

Reply:
Yes that's correct it's to handle updating an edge since updating an edge shouldn't count as a new edge